Exec/Make.Castro

```diff
@@ -373,12 +448,24 @@ ifeq ($(USE_HYPRE), TRUE)
 endif
 
 ifeq ($(USE_HDF5), TRUE)
-  INCLUDE_LOCATIONS += $(HDF5_DIR)/include
-  INCLUDE_LOCATIONS += $(HDF5_INCL)
-  LIBRARY_LOCATIONS += $(HDF5_DIR)/lib
-  LIBRARIES += -lhdf5 -lhdf5_fortran -lhdf5 -lz
+  DEFINES += -DBL_HDF5
+  INCLUDE_LOCATIONS += $(HDF5_HOME)/include 
+  ifneq ($(HOST), manda)
```

Comment:
I don't think we want specific machines hardcoded into the general make file

Reply:
This section has two machines hardcoded: `manda` and someone's Mac judging by the `apple-darwin` in the path.

I will remove these. Presumably this can be done by manually adding to `INCLUDE_LOCATIONS` and `LIBRARIES` on the `make` line.

Reply:
Done in 97ad4d5078f18da302b2407f9eb8b7bb14d32081